content/blog/2019/cve-2019-12995/index.md

```diff
@@ -0,0 +1,96 @@
+A bug in Istio’s JWT validation filter causes Istio Proxy to crash in certain cases when the request contains a malformed JWT token. The bug was discovered and reported by a user [on GitHub](https://github.com/istio/istio/issues/15084) on June 23, 2019.
+
+This bug affects all versions of Istio that are using the JWT authentication policy.
+
+The symptoms of the bug is an HTTP 503 error seen by the client, and
+
+{{< text text >}}
+Epoch 0 terminated with an error: signal: segmentation fault (core dumped)
+{{< /text >}}
+
+in the Envoy logs.
+
+The Envoy crash can be triggered using a malformed JWT without a valid signature, and on any URI being accessed regardless of the trigger\_rules in the JWT specification. Thus, this bug makes the Istio proxy vulnerable to a potential DoS attack.
```

Comment:
The trigger\_rule is probably incorrect.

Reply:
Some markdown linter do not like the _ to not be escaped, I wasn't sure about the markdown interpreter we're using. What would be the correct approach? `trigger_rules` ?